packages/docs-site/src/components/presenters/text-card/text-card.scss

```diff
@@ -0,0 +1,78 @@
+@import "@royalnavy/css-framework";
+
+$border-radius: 4px;
+$border-color: #dae2ec;
+$card-background-color: #f0f4f8;
+
+.text-card {
+  display: flex;
+  flex-direction: column;
+  color: #253b5b;
+}
+
+.text-card__head {
+  background-color: transparent;
+  padding: spacing(4) spacing(4) 0;
+  font-size: font-size(xl);
+  font-weight: 600;
+}
+
+.text-card__body {
+  padding: 0 spacing(4) spacing(4);
+}
+
+.text-card__body {
+  font-size: font-size(s);
+}
+
+.text-card__text {
+  font-size: font-size(s);
+}
+
+.text-card__link {
+  text-decoration: none;
+  font-weight: 600;
+  color: inherit;
+
+  svg {
```

Comment:
style using classname?

Reply:
I've done the same here in a couple of components. Lest have a chat about this and then I'll maybe change them all at once. Because the element is so tightly coupled to the parent i've styled it without a class.